src/api/app/components/watchlist_component.html.haml

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
This extra line can be removed.

```suggestion
```

Reply:
You're better than the linters :smile: thanks!

Reply:
I wonder why it's not caught by linters to be honest... loll